TuviSRPLib/TuviSRPLib/ProtonSRPUtilities.cs

```diff
@@ -78,8 +78,17 @@ public static BigInteger CalculateX(IDigest digest, BigInteger N, byte[] salt, b
 
         public static byte[] GetMailboxPassword(byte[] password, byte[] salt)
         {
```

Comment:
@al-kau , could you please add more comments to this code? Something like this: Blowfish doesn't support passwords longer than 72 bytes. Proton allows longer passwords but during hashing it ignores these extra symbols. So we do the same.

Reply:
Done ([look at here](https://github.com/Eppie-io/TuviSRPLib/pull/48/commits/bdd908d326ab5e42d7117c953297242c31f4cfb1#diff-87904d5eba6ed14c847336eca0959c7d944642d734523252272b2aa90f36299aR81))